src/services/api/travelPointsService.js

```diff
@@ -0,0 +1,16 @@
+import {travelPointsAxios} from "./axios";
+
+//USER
```

Comment:
I think this comment is redundant because the method name is self explanatory and all methods for USER will be self explanatory also in the future.